src/sentry/api/endpoints/assistant.py

```diff
@@ -33,11 +35,14 @@ class AssistantEndpoint(Endpoint):
 
     def get(self, request):
         """Return all the guides the user has not viewed or dismissed."""
```

Comment:
Update the comment since we're returning all guides now

Reply:
updated